terraform/plan.go

```diff
@@ -77,7 +79,9 @@ func (p *Plan) extractProviders() (map[string]Provider, error) {
 			if err != nil {
 				return nil, err
 			}
-			providers[name] = prov
+			if prov != nil {
+				providers[name] = prov
+			}
```

Comment:
Would be better maybe for this function to check?

```
if len(providers) == 0 {
		return nil, ErrNoProviders
	}
```

So then it's centralized

Reply:
Could be done at the end eventually yeah